plf-tomcat-resources/src/main/resources/conf/server.template.xml

```diff
@@ -192,6 +192,11 @@
                prefix="localhost_access_log" suffix=".txt"
                pattern="%h %l %u %t &quot;%r&quot; %s %b" />
         -->
+
+        <!-- Configure the remote ip valve for a reverse proxy on the local machine 
+             Adjust the internalProxies value with the ips of the reverse proxies if different.
+        -->
+        <Valve className="org.apache.catalina.valves.RemoteIpValve" internalProxies="127.0.0.1" remoteIpHeader="x-forwarded-for" proxiesHeader="x-forwarded-by" protocolHeader="x-forwarded-proto" />
```

Comment:
Shoudn't we leave the default value for internalProxies ? The default value includes more IPs, which could probably be useful in test environments.

Reply:
good catch, we should remain as generic as possible.